pom.xml

```diff
@@ -13,6 +13,17 @@
 		<neo4j.version>${env.NEO4JVERSION}</neo4j.version>
 	</properties>
 
+	<repositories>
+		<repository>
+			<snapshots>
+				<enabled>false</enabled>
+			</snapshots>
+			<id>neo4j-community</id>
+			<name>neo4j-community</name>
+			<url>http://m2.neo4j.com/community</url>
```

Comment:
@glindroth / @jennyowen can you confirm that this is the correct way to get pre-release jars?

Reply:
@eastlondoner no, m2.neo4j.org got closed down and we shouldn't use it any more. The community maven artifacts are here: https://bintray.com/neo4j/community/public

The site should have maven instructions but it's something like: 

```
                <repository>
                    <snapshots>
                        <enabled>false</enabled>
                    </snapshots>
                    <id>bintray-neo4j-community</id>
                    <name>bintray</name>
                    <url>https://neo4j.bintray.com/community</url>
                </repository>
```

I'm not sure we want to be introducing more very specific maven dependencies to the tests though? What do you need this for? Or is this PR so you can investigate the plugin test failure and the pom change isn't going to be in the final merge?

Reply:
@jennyowen I haven't introduced and new dependencies.

Neo4j has decided not to publish pre release versions of neo4j jars _that this project has dependencies on_ to Maven Central. This is a consequence of that.

This change is required to make this repository function if NEO4JVERSION is set to a version that is present on the community binary and not in Maven Central.